File: Daily Challanges/2026-02-07.js
function getLandingStance(startStance, rotation) {
  const trimmed_rotation = Math.abs(rotation) % 360

  if(trimmed_rotation < 180){
    return startStance;
  }else{
    return startStance == "Goofy" ? "Regular" : "Goofy";
  }
}

const test1 = getLandingStance("Regular", 90);
const test2 = getLandingStance("Regular", 180);
const test3 = getLandingStance("Goofy", -270);
const test4 = getLandingStance("Regular", 2340);
const test5 = getLandingStance("Goofy", 2160);

const check1 = "Regular";
const check2 = "Goofy";
const check3 = "Regular";
const check4 = "Goofy";
const check5 = "Goofy";

console.log(`Test1: ${test1 === check1}`);
console.log(`Test2: ${test2 === check2}`);
console.log(`Test3: ${test3 === check3}`);
console.log(`Test4: ${test4 === check4}`);
console.log(`Test5: ${test5 === check5}`);

File: Daily Challanges/2026-02-07.py
def get_landing_stance(start_stance, rotation):
    trimmed_rotation = abs(rotation) % 360

    if trimmed_rotation < 180:
        return start_stance
    else:
        return "Regular" if start_stance == "Goofy" else "Goofy"


test1 = get_landing_stance("Regular", 90)
test2 = get_landing_stance("Regular", 180)
test3 = get_landing_stance("Goofy", -270)
test4 = get_landing_stance("Regular", 2340)
test5 = get_landing_stance("Goofy", 2160)

check1 = "Regular"
check2 = "Goofy"
check3 = "Regular"
check4 = "Goofy"
check5 = "Goofy"

print(f"Test1: {test1 == check1}")
print(f"Test2: {test2 == check2}")
print(f"Test3: {test3 == check3}")
print(f"Test4: {test4 == check4}")
print(f"Test5: {test5 == check5}")